test/unit/math/fwd/mat/fun/quad_form_test.cpp

```diff
@@ -118,6 +119,47 @@ TEST(AgradFwdMatrixQuadForm, quad_form_sym_mat_fd) {
   EXPECT_FLOAT_EQ(810, resd(1,1).d_);
 }
 
+TEST(AgradFwdMatrixQuadForm, quad_form_sym_mat_dfd) {
+  using stan::math::quad_form_sym;
+  using stan::math::matrix_fd;
+  
+  Eigen::Matrix<double, -1, -1> ad(4,4);
+  matrix_fd bd(4,2);
+  
+  bd << 100, 10,
+  0,  1,
+  -3, -3,
+  5,  2;
+  ad << 2.0,  3.0, 4.0,   5.0, 
+  3.0, 10.0, 2.0,   2.0,
+  4.0,  2.0, 7.0,   1.0,
+  5.0,  2.0, 1.0, 112.0;
+  
+  // fvar<double> - fvar<double>
+  matrix_fd resd = quad_form_sym(ad,bd);
+}
```

Comment:
these are all just examples that I can instantiate something that couldn't be instantiated; these tests won't compile in current develop. I'll add EXPECT_NO_THROW(check_not_nan(resd)) tests in all of these